plugin/fzf.vim

```diff
@@ -455,7 +463,8 @@ function! s:pushd(dict)
       return 1
     endif
     let a:dict.prev_dir = cwd
-    execute 'lcd' s:escape(a:dict.dir)
+    let dir = s:escape(a:dict.dir)
+    execute 'lcd' has('win32unix') ? s:escape(s:cygpath(dir)) : dir
```

Comment:
Seems to be escaping twice (`s:escape(s:cygpath(s:escape(a:dict.dir)))`), intentional?